### DISCUSSION TITLE
TinyEngine中怎么实现监听store中的变量?

### DISCUSSION THREAD

**Discussion by stwflyfox** [opentiny/tiny-engine#328] [Q&A]
在TinyEngine中如何实现之前在 vue 代码实现的一个应用场景 ：在当前页面中监听store中的某个变量，然后在别的页面变更了这个变量值， 进而触发了当前页面的监听并做相应处理。

**Comment by chilingling** [comment]
页面管理 -> 页面生命周期
![image](https://github.com/opentiny/tiny-engine/assets/26962197/02975a0c-fb8a-4173-9a77-594967d7439f)

![image](https://github.com/opentiny/tiny-engine/assets/26962197/e09fee6a-17f7-4a89-a503-42cb14a4978b)

可以在这里添加 watch

**Comment by stwflyfox** [comment]
谢谢，可以实现了。
